docs/operation/cluster_and_service_configuration.rst

```diff
@@ -352,15 +352,27 @@ For example, to change the threshold for the disk space alert
                warning:
                  available: 10
 
-The new configuration must then be applied with Salt.
+Enable Prometheus Admin API
+"""""""""""""""""""""""""""
 
-.. parsed-literal::
+For security reasons, Prometheus Admin API is disabled by default.
+It can be enabled with the following:
 
-   root\@bootstrap $ kubectl exec --kubeconfig /etc/kubernetes/admin.conf \\
-                      -n kube-system -c salt-master salt-master-bootstrap -- \\
-                      salt-run state.sls \\
-                      metalk8s.addons.prometheus-operator.deployed \\
-                      saltenv=metalk8s-|version|
```

Comment:
Why would this be removed?

Reply:
Because there is already a section explaining how to apply the configuration.

Reply:
Maybe add a link to this section, as a reminder.